keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -299,37 +302,26 @@ provisioner: [1]
 supportedProtocols: [2]
 accessMode: {"ro", "wo", "rw"} [3]
 releasePolicy: {"Delete", "Retain"} [4]
-objectBucketName: [5]
+bucketIdentifier: [5]
 secretRef: [6]
   name:
   namespace:
 parameters: string:string [7]
 ```
 
-1. `provisioner`: (Optional) The name of the driver. If supplied the driver container and sidecar container are expected to be deployed. If omitted the `secretRef` is required for static provisioning.
-1. `supportedProtocols`: (Optional) An array of protocols the associated object store supports (e.g. swift, s3, gcs, etc.). *Only* serves a descriptive purpose and is not verified.
+1. `provisioner`: The name of the driver. If supplied the driver container and sidecar container are expected to be deployed. If omitted the `secretRef` is required for static provisioning.
```

Comment:
Why are the two "(Optional)" tags deleted here but remain for `accessModes`?

Reply:
Protocol allow users to be certain of what they're getting.  Without the field, we have to assume that the class is just named in some way that would clue in users to the supported clients.

Provisioner is mandatory as well - the only time it would allowable to be undefined would be for default bucket classes, which I thought we agreed didn't make sense.  The reason they don't is because we decided we weren't going to provide any matching (by protocol, accesssMode, etc) to classes.